kdcproxy/parse_pyasn1.py

```diff
@@ -100,6 +100,7 @@ def decode_proxymessage(data):
             realm = str(realm, "utf-8")
         except TypeError:  # Python 2.x
             realm = str(realm)
+        realm = realm.upper()
```

Comment:
Why are you doing this ?

Reply:
I now see your comment, realm is definitely a case sensitive name, even though Windows plays fast and lose with it. If you need a matching logic that is case -insensitive please do that. This change would break KDCs based on MIT kerberos or other more standard compliant Krb5 implementations.